packages/@react-aria/dnd/src/useDragHooks.ts

```diff
@@ -0,0 +1,23 @@
+import {DraggableCollectionOptions, DraggableCollectionState, useDraggableCollectionState} from '@react-stately/dnd';
+import {useMemo} from 'react';
+
+export interface DragHooks {
+    useDraggableCollectionState(props: DraggableCollectionOptions): DraggableCollectionState
+}
+
+export type DragHookOptions = Omit<DraggableCollectionOptions, 'collection' | 'selectionManager' | 'isDragging' | 'getKeysForDrag'>
```

Comment:
Open question: Is this what we want for options users can pass into useDragHook?